src/types/study-rooms.types.ts

```diff
@@ -0,0 +1,16 @@
+export type PageKey = 'current' | 'total';
+
+export type Page = Record<PageKey, number>;
```

Comment:
취향이지만 PageKey에 값이 얼마 없어서 이렇게 인라인해줄수도 있겠네요~
```suggestion
export type Page = Record<'current' | 'total', number>;
```